src/app/i18n/cs/resource.json

```diff
@@ -537,5 +537,6 @@
     "SET": "NASTAVIT",
     "CLR": "VYMAZAT",
     "Set RTS line status upon opening": "Nastavit stav řádky RTS při otevírání",
-    "Use RTS/CTS flow control": "Použijte řízení toku RTS/CTS"
+    "Use RTS/CTS flow control": "Použijte řízení toku RTS/CTS",
+    "Run macro (by id or name)": ""
```

Comment:
What should I do with translations? Is it better to use google-translate to try to approximate a translation, leaving it empty in the languages I do not know, or put the english text in the languages I do not know? 😅